runtime/browser/ui/gtk2_ui.h

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
+#define XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
+
+#include "ui/views/linux_ui/linux_ui.h"
+
+// Return the unique instance of class 'Gtk2UI'.
+extern views::LinuxUI* BuildGtk2UI();
+
+#endif  // XWALK_RUNTIME_BROWSER_UI_GTK2_UI_H_
```

Comment:
@halton 
My thought was that we might need to fork the "libgtk2ui" from Chromium in future as an independent library to add some own modifications, and the "gtk2_ui.h" is kept as the interface file of library. 
But, it's ok to drop this file right now. We can add it when there is a real need. 
The patch will be updated soon.